retail-managment-system/rms-ui/src/app/pages/transaction/transaction.component.ts

```diff
@@ -123,6 +124,7 @@ export class TransactionComponent implements OnInit {
         this.submit = false;
       }
     );
+    console.log(this.transactionPostForm.value)
```

Comment:
before pushing check if there any console or comment

Reply:
removed it